README.md

```diff
@@ -105,20 +105,97 @@ The Hugo community and maintainers are very active and helpful and the project b
 
 If you have any questions about how to contribute or what to contribute, please ask on the [forum](http://discuss.gohugo.io).
 
-## Code Contribution Guideline
+## Code Contribution Guidelines
 
 We welcome your contributions.
 To make the process as seamless as possible, we ask for the following:
 
 * Go ahead and fork the project and make your changes. We encourage pull requests to discuss code changes.
 * When you’re ready to create a pull request, be sure to:
-     * Sign the [CLA](https://cla-assistant.io/spf13/hugo)
+     * Sign the [CLA](https://cla-assistant.io/spf13/hugo).
      * Have test cases for the new code. If you have questions about how to do it, please ask in your pull request.
-     * Run `go fmt`
+     * Run `go fmt`.
+     * Follow the **Git Commit Message Guidelines** below.
      * Squash your commits into a single commit. `git rebase -i`. It’s okay to force update your pull request.
-     * **Write a good commit message.** This [blog article](http://chris.beams.io/posts/git-commit) is a good resource for learning how to write good commit messages, the most important part being that each commit message should have a title/subject in imperative mood starting with a capital letter and no trailing period: *"Return error on wrong use of the Paginator"*, **NOT** *"returning some error."* Also, if your commit references one or more GitHub issues, always end your commit message body with *See #1234* or *Fixes #1234*. Replace *1234* with the GitHub issue ID. The last example will close the issue when the commit is merged into *master*. Sometimes it makes sense to prefix the commit message with the packagename (or docs folder) all lowercased ending with a colon. That is fine, but the rest of the rules above apply. So it is "tpl: Add emojify template func", not "tpl: add emojify template func.", and "docs: Document emoji", not "doc: document emoji."
      * Make sure `go test ./...` passes, and `go build` completes. Our [Travis CI loop](https://travis-ci.org/spf13/hugo) (Linux and OS&nbsp;X) and [AppVeyor](https://ci.appveyor.com/project/spf13/hugo/branch/master) (Windows) will catch most things that are missing.
 
+### Git Commit Message Guidelines
+
+Quality Git commit messages are critical to a large project;
+therefore, we've established the following guidelines:
+
+1. Prefix the subject with the primary affected **package**.
+1. After the package prefix, **capitalize** the subject.
+1. End the subject **without punctuation**.
+1. Use the **imperative** mood in the subject and body.
```

Comment:
No sure what Chris Beams says a bout the body and mood, but this will be impossible to enforce.